sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawNetwork.java

```diff
@@ -3,30 +3,31 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
-import java.util.Map;
+import java.util.stream.Collectors;
 
 /**
  * Raw YAML network.
  */
 public class RawNetwork {
     private final List<Integer> hostPorts;
     private final List<Integer> containerPorts;
-    private final Map<String, String> labels;
+    private final String labelsCsv;
 
     @JsonCreator
     private RawNetwork(
             @JsonProperty("host-ports") List<Integer> hostPorts,
```

Comment:
Yes, that's why it's a `Map` in `DefaultNetworkSpec`. The current (IMO, bad) yaml uses this:
```
networks:
   mesos-bridge:
        host-ports: [1025]
        container-ports: [80]
```
Thus the lists. I can change it in a another PR after input from product, but I think it's not worth re-resting this PR because it's not fully supported capability.